api/PointerEvent.json

```
@@ -492,28 +492,48 @@
             "description": "Fractional coordinates for <code>mouse</code>.",
             "support": {
               "chrome": {
-                "version_added": "64"
+                "version_added": "64",
+                "partial_implementation": true,
+                "notes": "<code>movementX</code> and <code>movementY</code> are not fractional, see <a href='https://crbug.com/802067'>Chromium bug 802067</a>."
               },
               "chrome_android": {
-                "version_added": "64"
-              },
-              "edge": {
-                "version_added": "≤79"
+                "version_added": "64",
+                "partial_implementation": true,
+                "notes": "<code>movementX</code> and <code>movementY</code> are not fractional, see <a href='https://crbug.com/802067'>Chromium bug 802067</a>."
               },
+              "edge": [
+                {
+                  "version_added": "79",
+                  "partial_implementation": true,
+                  "notes": "<code>movementX</code> and <code>movementY</code> are not fractional, see <a href='https://crbug.com/802067'>Chromium bug 802067</a>."
+                },
+                {
+                  "version_added": "12",
+                  "version_removed": "79",
+                  "partial_implementation": true,
+                  "notes": "Only <code>clientX</code>, <code>clientY</code>, <code>pageX</code> and <code>pageY</code> are fractional."
+                }
+              ],
               "firefox": {
-                "version_added": null
+                "version_added": false
               },
               "firefox_android": {
                 "version_added": null
```

If Firefox Desktop doesn't support it, I'm confident Firefox Android wouldn't support it either.

```suggestion
                "version_added": false
```